src/votekit/elections/election_types.py

```diff
@@ -1117,3 +1120,317 @@ def __init__(
             seats=seats,
             tiebreak=tiebreak,
         )
+
+
+class RandomDictator(Election):
+    """
+    Choose a winner randomly from the distribution of first place votes. For multi-winner elections
+    repeat this process for every winner, removing that candidate from every voter's ballot
+    once they have been elected.
```

Comment:
For multi-winner, sample without replacement from FPV distribution?

Reply:
I'm going to leave this as is for now and take some more time to think about a fix